Modules/mathmodule.c

```diff
@@ -3450,6 +3450,49 @@ math_perm_impl(PyObject *module, PyObject *n, PyObject *k)
     return NULL;
 }
 
+/* least significant 64 bits of the odd part of factorial(n), for n in range(68). */
```

Comment:
Perhaps expand the comment with  the pure python code to generate the table:

```
for n in range(68):
    f = factorial(n)
    f //= f & -f
    f %= 2 ** 64
    print(f'0x{f:016x}')
```

Reply:
Done.